typescript/src/hello/world.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { equal } from "assert";
+import world from './world';
```

Comment:
What's the point of the extra directory hierarchy (i.e., why "hello/")?

Reply:
Was just to demonstrate alternative. Will remove.